### PR TITLE
feat: add method for decoding sealed results

### PR DESCRIPTION
**Pull request by TheUnderScorer**

### CODE REVIEW

File: .swagger-codegen-ignore [hunk 1]
```diff
@@ -28,3 +28,4 @@ git_push.sh
 tox.ini
 test/*.py
 fingerprint_pro_server_api_sdk/models/many_requests_response.py
+requirements.txt
```
Comment (ilfa) [inline .swagger-codegen-ignore:31]: Let's add `test-requirements.txt` too and remove templates

File: fingerprint_pro_server_api_sdk/sealed.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+
+class DecryptionKey:
+    def __init__(self, key, algorithm):
+        self.key = key
+        self.algorithm = algorithm
+
+
+class UnsealError(Exception):
+    exception: Exception
```
Comment (ilfa) [inline fingerprint_pro_server_api_sdk/sealed.py:21]: This way to define types wouldn't work for Python 2.7, 3.4, and 3.5

File: fingerprint_pro_server_api_sdk/sealed.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    errors = []
+    for decryption_key in decryption_keys:
+        if decryption_key.algorithm == DecryptionAlgorithm['Aes256Gcm']:
+            try:
+                return unseal_aes256gcm(sealed_data, decryption_key.key)
+            except Exception as e:
+                errors.append(UnsealError(e, decryption_key))
+                continue
+        else:
+            raise ValueError(f"Unsupported decryption algorithm: {decryption_key.algorithm}")
```
Comment (ilfa) [inline fingerprint_pro_server_api_sdk/sealed.py:62]: This f prefix wouldn't work for Python 2.7, 3.4, and 3.5

**Review by ilfa — CHANGES_REQUESTED** [review]
We need to decide something about support of the deprecated python versions first

File: .github/workflows/test.yml [hunk 4]
```diff
@@ -13,11 +13,11 @@ jobs:
 
     strategy:
       matrix:
-        python-version: [ "3.6", "3.7", "3.8", "3.9", "pypy3" ]
+        python-version: [ "3.6", "3.7", "3.8", "3.9", "pypy3.10" ]
```
Comment (ilfa) [inline .github/workflows/test.yml:16]: Could you please add the missing version we support from [this list](https://devguide.python.org/versions/)?

Comment (ilfa) [inline .github/workflows/test.yml:16]: Could you please add the missing version we support from [this list](https://devguide.python.org/versions/)?

**Review by ilfa — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 4 above)
Comment (TheUnderScorer) [reply .github/workflows/test.yml]: Done!

**Review by ilfa — APPROVED** [review]

**Pull request merged**